packages/devtools_app/lib/src/tree.dart

```diff
@@ -124,23 +149,27 @@ class _TreeViewItemState<T extends TreeNode<T>> extends State<TreeViewItem<T>>
   Widget build(BuildContext context) {
     return Padding(
       padding: EdgeInsets.only(left: nodeIndent(widget.data)),
-      child: Row(
-        mainAxisSize: MainAxisSize.min,
-        children: [
-          widget.data.isExpandable
-              ? InkWell(
-                  onTap: _onPressed,
-                  child: RotationTransition(
-                    turns: expandArrowAnimation,
-                    child: Icon(
-                      Icons.arrow_drop_down,
-                      size: defaultIconSize,
+      child: Container(
+        color:
+            widget.data.isSelected ? Theme.of(context).selectedRowColor : null,
+        child: Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            widget.data.isExpandable
+                ? InkWell(
+                    onTap: _onExpanded ?? _onSelected,
```

Comment:
_onExpanded doesn't look like it will ever be null since it is a private method in this class. I would think that we would want to default to _onSelected and have _onSelected call widget.onItemExpanded if the node is expandable

Reply:
Ah yeah, this isn't right. However, I do think that we should keep both `_onExpanded` and `_onSelected` as we can have slightly different expansion behaviors depending on whether you tap on the expansion arrow or the row itself. For example, in this PR I've set the tree up so that selecting a node will expand it but won't collapse if it's already expanded, making the expansion arrow the only way to collapse subtrees.